src/char/custom/CustomCharDb.py

```
@@ -148,11 +206,25 @@ def _save_locked(self):
             with open(temp_path, "w", encoding="utf-8") as file:
                 json.dump(self._data, file, indent=4, ensure_ascii=False)
             os.replace(temp_path, self.db_path)
+            return True
         except Exception as error:
             if self.logger:
                 self.logger.error("Failed to save custom char DB", error)
```

Ruff (0.16.1)

[error] Too many arguments for `logging` format string

(PLE1205)